yarn-project/noir-contracts/src/contracts/pending_commitments_contract/src/main.nr

```diff
@@ -165,54 +138,64 @@ contract PendingCommitments {
         owner: Field,
         insert_fn_selector: Field,
         get_then_nullify_fn_selector: Field,
-        get_note_zero_fn_selector: Field,
+        get_note_zero_fn_selector: Field
     ) {
         // nested call to create/insert note
-        let _callStackItem1 = context.call_private_function(inputs.call_context.storage_contract_address, insert_fn_selector, [amount, owner]);
+        let _callStackItem1 = context.call_private_function(inputs.call_context.storage_contract_address,
+            insert_fn_selector,
+            [amount, owner]);
         // nested call to read and nullify that note
-        let _callStackItem2 = context.call_private_function(inputs.call_context.storage_contract_address, get_then_nullify_fn_selector, [amount, owner]);
-         // nested call to confirm that balance is zero
-        let _callStackItem3 = context.call_private_function(inputs.call_context.storage_contract_address, get_note_zero_fn_selector, [owner]);
+        let _callStackItem2 = context.call_private_function(inputs.call_context.storage_contract_address,
+            get_then_nullify_fn_selector,
+            [amount, owner]);
+        // nested call to confirm that balance is zero
+        let _callStackItem3 = context.call_private_function(inputs.call_context.storage_contract_address,
+            get_note_zero_fn_selector,
+            [owner]);
```

Comment:
@f01dab1e In this case doing this:
```
let _callStackItem3 = context.call_private_function(
    inputs.call_context.storage_contract_address,
    get_note_zero_fn_selector,
    [owner]
);
```
would be nicer but it's a detail.

Reply:
sent a fix https://github.com/noir-lang/noir/pull/3526